src/mmw/mmw/settings/layer_settings.py

```diff
@@ -135,7 +135,7 @@
     },
     {
         'code': 'drb_streams_v1',
-        'display': 'DRB Stream Network',
+        'display': 'Delaware River Basin High Resolution Stream Network',
```

Comment:
Ah! Likely true. Let me try that and verify that it works, as it's much simpler than my alternative here.